docs/guide/units.rst

```diff
@@ -0,0 +1,110 @@
+.. _units-sunpy:
+
+Units
+*****
+
+This section of the guide talks about representing physical units sunpy.
+sunpy makes use of the `astropy.units` for this task.
+
+Quantity objects
+================
+All functions in sunpy that accept or return numbers associated with physcial quantities accept and return `~astropy.units.Quantity` objects.
+These objects represent a number (or an array of numbers) and a unit.
+This means sunpy is always explicit about the units associated with a value.
+Quantities and units are powerful tools for keeping track of variables with a physical meaning and make it straightforward to convert the same physical quantity into different units.
+
+To use units we must first import them from Astropy.
+To save on typing it's standard practice to import the units module as ``u``::
+
+   >>> import astropy.units as u
+
+Once we have imported the units module we can create a `~astropy.units.Quantity` by multiplying a plain number by a unit::
```

Comment:
```suggestion
Once we have imported the units module we can create a `~astropy.units.Quantity` by multiplying an integer by a unit::
```
`plain number` is a weird term here - maybe `integer`?

Reply:
I'll change this to just "number", as I think `integer` could have the unintended effect of implying you can't multiply a floating point number here (when you can).